website/docs/releases/2024/v2024.5.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Release 2024.5

```

Comment:
Nice try, but we'll probably not release in May. We usually use `2024.next` until we know the release month